README.md

```diff
@@ -1 +1,219 @@
+
+Bloock Identity follows the [Verifiable Credentials Data Model v2.0](https://www.w3.org/TR/vc-data-model-2.0/) standard and [Decentralized Identifiers (DIDs) v1.0](https://www.w3.org/TR/did-core/) standard.
+
+## The basics
+
+### What do we mean by Identity?
+An identity can be a person, a company, an organization, a DAO, or a government. Identity can even be a thing: a chair, a room, a bot, and so on. When we talk about identities, we are referring to "identities as accounts".
+
+### What is a claim?
+An Identity can provide a claim. You can think of a claim as a statement: something an Identity is saying.
+
+Most of the time, these statements refer to other identities. In other words, **claims usually create relations between identities.**
+
+For example, when a university (identity) says that a student (identity) has a degree, this is a statement (claim) about the student. This statement creates a relation between the student and the university.
+
+### What are the agents in this process?
+- **Issuer**: An actor who makes a claim.
+- **Holder**: An actor who receives a claim.
+- **Verifier**: An actor who verifies if the content of a claim is issued by a specific identity and held by another specific identity.
+
+### And therefore, what is a verifiable credential?
+**Credential**: Data that is needed to prove that a claim is issued by a specific identity and held by another specific identity. This data is composed of **a claim and a proof**.
+
+## Decentralized Identifiers (DIDs)
+
+A [DID](https://www.w3.org/TR/did-core/#dfn-decentralized-identifiers) is a simple text string consisting of three parts: 
+1. The `did` URI scheme identifier.
+2. The identifier for the [DID method](https://www.w3.org/TR/did-core/#dfn-did-methods).
+3. The DID method-specific identifier.
+
+![https://cdn.discordapp.com/attachments/1151100469490487307/1206967003500511343/Group_1165.png?ex=65ddee9d&is=65cb799d&hm=ada436c44d4996110fa3241c212b26dfd610a9e94c1d9015600ff9d144708c8e&](https://cdn.discordapp.com/attachments/1151100469490487307/1206967003500511343/Group_1165.png?ex=65ddee9d&is=65cb799d&hm=ada436c44d4996110fa3241c212b26dfd610a9e94c1d9015600ff9d144708c8e&)
+Currently in Bloock Identity you can use the following methods.
+- [did:polygonid](https://github.com/0xPolygonID/did-polygonid)
+- Soon they will be extended...
+
+## did:polygonid
+
+The method id called `did:polygonid` has the following characteristics.
+- Is generated from three identity trees and controlled by [Baby JubJub keys](https://docs.iden3.io/publications/pdfs/Baby-Jubjub.pdf). Therefore this key you can create with [Bloock Keys product](go_to_baby_jub_jub_keys documentation) in a managed way or by yourself locally. It will be very important to be able to control your identity.
```

Comment:
It's derived from a Baby JubJub key and other cryptographic material. This key can be created with [Bloock Keys product](go_to_baby_jub_jub_keys documentation) in a managed way or by yourself locally. It will be very important to be able to control your identity.

Posar link a documentació de iden/polygon per més info de la generació del did